docker-compose-tests.yml

```diff
@@ -15,17 +15,16 @@ services:
     links:
       - gp-finder-db:es
 
-  travis-tests:
+  ci-tests:
```

Comment:
Can this be renamed to `test-ci` - that is what I've done across the rest of the repos.